### DISCUSSION TITLE
More options for text formatting

### DISCUSSION THREAD

**Discussion by mgogoulos** [withlogicco/django-prose#31] [Q&A] [answered]
Hi, while the basic text formatting options are there, I noticed that there are some notable ones missing, for example

1. Resize image
2. More options for Header
3. Font/color selector

Is there a way to add them? Or Trix editor does not even support them?

Thanks and regards

**Comment by parisk** [comment, marked as answer]
Hey @mgogoulos, 

Thanks for your comment!

The stock Trix we are using does not support these options. One way to do this would be overriding the widget with your own HTML[^1], CSS[^2] and JavaScript[^3].

Trix is quite configurable so you should be able to do this, although I presume we will make Django Prose more customisable in the future, if there is need for that.

[^1]: https://github.com/withlogicco/django-prose/blob/73ac344800d0b1f7a95ca506abce01158fe74710/prose/templates/prose/forms/widgets/editor.html
[^2]: https://github.com/withlogicco/django-prose/blob/73ac344800d0b1f7a95ca506abce01158fe74710/prose/widgets.py#L9-L12
[^3]: https://github.com/withlogicco/django-prose/blob/73ac344800d0b1f7a95ca506abce01158fe74710/prose/widgets.py#L14-L17

**Reply by mgogoulos** [reply]
Still somehow surprised that Trix does not have these options out of the box, perhaps to keep the options list very minimal. Will use your suggestions if I need to add them, for now the base options work well!
